presidio-image-redactor/tests/integration/test_dicom_image_pii_verify_engine_integration.py

```diff
@@ -0,0 +1,130 @@
+"""Integration test for dicom_image_pii_verify_engine
+
+Note we are not checking exact pixel data for the returned image
+because that is covered by testing of the "verify" function in
+the original parent ImagePiiVerifyEngine class.
+"""
+import PIL
+import pydicom
+import json
+import os
+
+from presidio_image_redactor.dicom_image_pii_verify_engine import (
+    DicomImagePiiVerifyEngine,
+)
+import pytest
+
+SCRIPT_DIR = os.path.dirname(__file__)
+PADDING_WIDTH = 25
+
+
+@pytest.fixture(scope="function")
+def mock_engine():
+    """Instance of the DicomImagePiiVerifyEngine"""
+    # Arrange
+
+    # Act
+    dicom_image_pii_verify_engine = DicomImagePiiVerifyEngine()
+
+    return dicom_image_pii_verify_engine
+
+
+@pytest.fixture(scope="module")
+def mock_instance():
+    """DICOM instance to use in testing"""
+    # Assign
+    filepath = f"{SCRIPT_DIR}/resources/0_ORIGINAL.dcm"
+
+    # Act
+    instance = pydicom.dcmread(filepath)
+
+    return instance
+
+
+@pytest.fixture(scope="module")
+def mock_results():
```

Comment:
If you use the same fixtures in multiple tests/files, consider creating them in the `conftest.py` file with a broader scope to avoid code duplication and to simplify the flow

Reply:
Since this fixture is not used across different files, I will keep this in here. However, I will keep this in mind for future tests.

Reply:
I do see that `mock_instance` is used across files so I will move that to `conftest.py`.